uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -40,9 +40,9 @@ Feature: Greengrass V2 LogManager
         """
         And I deploy the Greengrass deployment configuration
         Then the Greengrass deployment is COMPLETED on the device after 2 minutes
+        Then I verify the aws.greengrass.LogManager component is RUNNING using the greengrass-cli
         Then I verify that it created a log group for component type GreengrassSystemComponent for component System, with streams within 120 seconds in CloudWatch
         And I verify that it created a log group for component type UserComponent for component UserComponentA, with streams within 120 seconds in CloudWatch
-        And I verify the rotated files are not deleted except for the active log file for component UserComponentA
```

Comment:
This step did not make sense and it does not exist on EF

Reply:
we don't want to accidentally delete the files (bug in the code for any reason) when the delete files on upload is set to true. Are we only writing the steps that exist on EF?

Reply:
This step was not on EF + this step doesn't make sense in the context of what this test is testing, the log manager should **never** delete the active log file. We could improve this step by adding it and making it correct but I want us to focus first on porting what we have and then improving them

Reply:
There are other tests that verify files btw